core/src/main/java/com/linecorp/armeria/internal/AnnotatedHttpServiceParamUtil.java

```diff
@@ -310,10 +312,10 @@ private static String getName(Object element) {
         if (element instanceof Parameter) {
             final Parameter parameter = (Parameter) element;
             checkArgument(parameter.isNamePresent(),
-                          "Cannot obtain the name of the parameter or field automatically. " +
+                          "cannot obtain the name of the parameter or field automatically. " +
                           "Please check whether your compile option includes '-parameters'. " +
-                          "If not, you need to specify parameter and header names with @" +
-                          Param.class.getName() + " and @" + Header.class.getName() + '.');
+                          "If not, you need to specify parameter and header names with @%s and @%s.",
+                          Param.class.getName(), Header.class.getName());
```

Comment:
Could we use traditional `if` block instead of `checkArgument()` here, because otherwise `Class.getName()` will be invoked even for successful path.